frontend/views/components/CopyToClipboard.vue

```diff
@@ -0,0 +1,107 @@
+<template lang='pug'>
+.c-wrapper(@click='copyToClipboard')
+  input.c-invisible-input(
+    type='text'
+    ref='input'
+    :value='textToCopy'
+  )
+  slot
+  transition(name='fade')
+    i18n.c-tooltip(
+      tag='p'
+      v-if='ephemeral.tooltipActive'
+    ) Copied to clipboard!
+</template>
+
+<script>
+import Tooltip from '@components/Tooltip.vue'
+
+export default {
+  name: 'copyToClipboard',
+  data () {
+    return {
+      ephemeral: {
+        tooltipActive: false
+      }
+    }
+  },
+  components: {
+    Tooltip
+  },
+  props: {
+    textToCopy: {
+      type: String,
+      required: true
+    }
+  },
+  methods: {
+    copyToClipboard () {
+      // if the user is using the device that supports web share API, use it and then skip the other logics below.
+      if (navigator.share) {
+        this.activateWebShare()
+        return
+      }
+
+      this.$refs.input.select()
```

Comment:
shouldn't you use v-focus? (autofocus.js)

Reply:
I thought `v-focus` was not the use case here because the invisible input element here should only be selected when the user clicks on the invite-link whereas `v-focus` (from what I understand) makes it automatically focused when the input is first shown on the page. (Not so sure if this makes sense to you but would you tell me your idea if it doesn't?)